### DISCUSSION TITLE
LiDAR Support

### DISCUSSION THREAD

**Discussion by jeffreyianwilson** [alicevision/Meshroom#2155] [General]
Are there any plans to add lidar/structured/unstructured point cloud support?

**Comment by fabiencastan** [comment]
Yes, we would like to do it, but it's not yet on the top of the long todo list!
We plan to work on it in 2024.
Any contribution on the topic would be highly appreciated.

**Comment by natowi** [comment]
Reading/Writing should not be a problem, there is https://pdal.io/en/latest/stages/readers.html for example.
But we would need some alignment tools (manual via UI and automated)

**Comment by JoeK60** [comment]
Would it be possible to cut out a section of a LiDAR Point Cloud, maybe manually re-format the data to a specific known data format Meshroom recognizes, and then have Meshroom complete a model of the data?  For example - a building that has been scanned from all directions?

I'm just trying to explore a workflow to see if data can be input into Meshroom without the data being photos.  Doable or too many hurdles?
Thanks.

**Comment by natowi** [comment]
See  https://github.com/alicevision/AliceVision/pull/1668 https://github.com/alicevision/AliceVision/pull/1652
(not yet in the current build)
